scenes/game/local_game.gd

```diff
@@ -3053,13 +3089,33 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 
 	return events
 
+func handle_player_draw_or_discard_to_effect(performing_player : Player, card_id, effect):
+	var events = []
+	var target_hand_size = effect['amount']
+	var hand_size = performing_player.hand.size()
+	if hand_size < target_hand_size:
+		var amount_to_draw = target_hand_size - hand_size
+		events += performing_player.draw(amount_to_draw)
+	elif hand_size > target_hand_size:
+		var amount_to_discard = hand_size - target_hand_size
+		var discard_effect = {
+			"effect_type": "self_discard_choose",
+			"amount": amount_to_discard
+		}
+		events += handle_strike_effect(card_id, discard_effect, performing_player)
+	else:
+		pass
+	return events
+
 func get_card_stat(check_player : Player, card : GameCard, stat : String) -> int:
 	var value = card.definition[stat]
 	if str(value) == "X":
 		if active_strike:
 			return check_player.strike_stat_boosts.strike_x
 		else:
 			assert(false, "ERROR: No support for interpreting X outside of strikes")
+	elif str(value) == "CARDS_IN_HAND":
+		value = check_player.hand.size()
```

Comment:
There's nothing that happens to print out cards' coded guard values or anything right? I was worrying about how to do things like merkava's distance from opponent's Xs but this is probably an easier solution than what I was thinking of

Reply:
There isn't. And if there is, it needs to account for these things.